app/models/concerns/host_common.rb

```diff
@@ -156,56 +150,42 @@ def hg_class_ids
   end
 
   def host_class_ids
-    is_a?(Host::Base) ? host_classes.pluck(:puppetclass_id) : []
+    debugger
+    throw 'Old code'
   end
 
   def all_puppetclass_ids
-    cg_class_ids + hg_class_ids + host_class_ids
+    debugger
+    throw 'Old code'
   end
 
   def classes(env = environment)
-    conditions = {:id => all_puppetclass_ids }
-    if env
-      env.puppetclasses.where(conditions)
-    else
-      Puppetclass.where(conditions)
-    end
+    debugger
+    throw 'Old code'
   end
 
   def puppetclass_ids
-    classes.reorder('').pluck('puppetclasses.id')
+    debugger
+    throw 'Old code'
   end
 
   def classes_in_groups
-    conditions = {:id => cg_class_ids }
-    if environment
-      environment.puppetclasses.where(conditions) - parent_classes
-    else
-      Puppetclass.where(conditions) - parent_classes
-    end
+    debugger
```

Comment:
There are a number of debugger statements in this class. Do you want to throw a StandardError with the message `Old code` or is this temporary?